src/appmake/ace-tap.c

```diff
@@ -128,7 +128,14 @@ int acetap_exec(char* target)
                 strncpy(name, blockname, 10);
             } else {
                 strcpy(name, blockname);
+#if __GNUC__ >= 8
```

Comment:
I have to admit that I don't like this since it only removes the diagnostic for one compiler.

 I think we'd be better off by using a snprintf() format string to right pad with spaces.

Reply:
I agree. The point is that I'm not a 'Warning' lover, since warnings are there to prevent problems, and letting them exist gives me an uncomfortable feeling. I like compiles to be crisp.

Thanks for the hint. I'll look into the snprintf() format solution.

Grtz,
PB